client/state/sharing/selectors.js

```diff
@@ -13,52 +13,61 @@ import { getCurrentUserId } from 'state/current-user/selectors';
 import { getSelectedSiteId } from 'state/ui/selectors';
 import { getKeyringConnectionsByName } from './keyring/selectors';
 import { getSiteUserConnectionsForService } from './publicize/selectors';
+import { getKeyringServiceByName } from './services/selectors';
 
 /**
  * Given a service, returns a flattened array of all possible accounts for the
  * service for which a connection can be created.
  *
- * @param  {Object} state   Global state tree
- * @param  {String} service The name of the service to check
- * @return {Array}          Flattened array of all possible accounts for the service
+ * @param  {Object} state       Global state tree
+ * @param  {String} serviceName The name of the service to check
+ * @return {Array}              Flattened array of all possible accounts for the service
  */
-export function getAvailableExternalAccounts( state, service ) {
+export function getAvailableExternalAccounts( state, serviceName ) {
```

Comment:
Yeah, external accounts refer to all "social" accounts external to WordPress.com, I agree it may be confusing with "additional external users" but it's used all over and makes sense, I'd rather not fix that here.